utils/queues/fair_sorted.go

```diff
@@ -5,36 +5,12 @@ import (
 	"encoding/json"
 	"fmt"
 	"strconv"
-	"time"
 
 	"github.com/gomodule/redigo/redis"
 	"github.com/nyaruka/gocommon/dates"
 	"github.com/nyaruka/gocommon/jsonx"
 )
 
-// Task is a wrapper for encoding a task
-type Task struct {
-	Type       string          `json:"type"`
-	OwnerID    int             `json:"-"`
-	Task       json.RawMessage `json:"task"`
-	QueuedOn   time.Time       `json:"queued_on"`
-	ErrorCount int             `json:"error_count,omitempty"`
-}
-
-// Priority is the priority for the task
-type Priority int
-
-const (
-	// HighPriority is the highest priority for tasks
-	HighPriority = Priority(-10000000)
-
-	// DefaultPriority is the default priority for tasks
-	DefaultPriority = Priority(0)
-
-	// LowPriority is the lowest priority for tasks
-	LowPriority = Priority(+10000000)
```

Comment:
we don't use this - only high/default as a way to push some things to the front of the queue